### PR TITLE
Minor Link Fix

### PR DESCRIPTION
**Pull request by abhi-bhatra**
Some links are either broken or no longer exist, cause 404 error. 

not found

Fixed the links with the right address.
1. 2-Regression/1-Tools/README.md
2. 2-Regression/4-Logistic/README.md
3. 5-Clustering/2-K-Means/README.md
4. 6-NLP/1-Introduction-to-NLP/README.md
5. 6-NLP/3-Translation-Sentiment/README.md
6. 6-NLP/5-Hotel-Reviews-2/README.md

### CODE REVIEW

File: 5-Clustering/2-K-Means/README.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ Hint: Try to scale your data. There's commented code in the notebook that adds s
 
 ## Review & Self Study
 
-Take a look at Stanford's K-Means Simulator [here](https://stanford.edu/class/engr108/visualizations/kmeans/kmeans.html). You can use this tool to visualize sample data points and determine its centroids. With fresh data, click 'update' to see how long it takes to find convergence. You can edit the data's randomness, numbers of clusters and numbers of centroids. Does this help you get an idea of how the data can be grouped?
+Take a look at K-Means Simulator [here](https://user.ceng.metu.edu.tr/~akifakkus/courses/ceng574/k-means/). You can use this tool to visualize sample data points and determine its centroids. You can edit the data's randomness, numbers of clusters and numbers of centroids. Does this help you get an idea of how the data can be grouped?
```
Comment (jlooper) [inline 5-Clustering/2-K-Means/README.md:245]: Can you edit this to "Take a look at a K-Means Simulator [such as this one](link)"

**Review by jlooper — CHANGES_REQUESTED** [review]
thank you for this tidy-up! Just a minor change, please

File: 5-Clustering/2-K-Means/README.md (hunk 1 above)
Comment (jlooper) [reply 5-Clustering/2-K-Means/README.md:245]: I'll fix it, thanks

**Review by jlooper — APPROVED** [review]

File: 5-Clustering/2-K-Means/README.md (hunk 1 above)
Comment (abhi-bhatra) [reply 5-Clustering/2-K-Means/README.md]: Thanks for the review Jen. I am changing this to the text you mentioned.

**Pull request merged**